dotcom-rendering/src/server/lib/get-content-from-url.js

```diff
@@ -1,6 +1,6 @@
 const fetch = require('node-fetch');
 
-async function getContentFromURL(_url) {
+async function getContentFromURL(_url, _headers) {
```

Comment:
Possibly worth updating the function name `getContentFromURLAndHeaders`?

Reply:
I think it’s best to add JSDoc comments. For example:

```ts
/**
 * Get DCR content from a `theguardian.com` URL.
 * Takes in optional `X-Gu-*` headers to send.
 */
async function getContentFromURL
```